packages/e2e-tests/specs/experiments/multi-entity-editing.test.js

```diff
@@ -124,6 +124,18 @@ const openEntitySavePanel = async () => {
 		await page.waitForSelector( '.entities-saved-states__panel' );
 	}
 	// If we made it this far, the panel is opened.
+
+	// Expand to view savable entities if necessary.
```

Comment:
```suggestion
	// Expand to view saveable entities if necessary.
```

Reply:
This is one of those things where the english language is stupid.  🤣  Typing 'savable' always feels so awkward, but 'saveable' always prompts red underline of incorrectness by spellcheckers.

'Savable' is oddly the proper spelling, but I 'saveable' is listed in certain places as a valid alternative.  Im more than happy to include the 'e' if thats what we want, I like it better but have been conforming to the spellcheck bias.

Reply:
Interesting, I didn't know there were two ways of spelling it. I don't mind then. Let's go with savable if it's what the spellchecker wants.